packages/util-rlp/test/getTests.ts

```diff
@@ -14,7 +14,7 @@ export default function getTests (file: string) {
     name,
     input: tests[name].in[0] !== '#'
       ? tests[name].in
-      : bnToU8a(new BN(tests[name].in.slice(1)), -1, true),
+      : bnToU8a(new BN(tests[name].in.slice(1)), { isLe: false, isNegative: false }),
```

Comment:
This one bugged me for 1h, I thought I f*cked up somewhere... until I check on the internet, Ethereum uses BE.

Reply:
100%